docs/application/dotnet/guides/nui/flexcontainer.md

```diff
@@ -19,9 +22,17 @@ The following example shows how to create a gallery-like flexbox layout using th
 
 ![Flex container example](media/flexbox-demo.jpg)
 
-1.  Create a flex container as the whole view and set its resize policy to fill its parent (the window):
+1.  Add namespace using:
 
+    ```cs
+    using Tizen.NUI;
+    using Tizen.NUI.BaseComponents;
+    using Tizen.NUI.UIComponents;
     ```
+
+2.  Create a flex container as the whole view and set its resize policy to fill its parent (the window):
+
+    ```cs
```

Comment:
Global.  Would you please refer https://github.com/Samsung/tizen-docs/blob/master/styleguide/style.md#supported-languages? It seems we should use "csharp", not "cs".